top-nav-drill/main.css

```diff
@@ -2,4 +2,80 @@
   margin: 0 auto;
 }
 
+section
+{
+
+  width: 60%;
+  margin: 0 auto;
+  
+}
+
+section section
+{
+
+  width: 100%;
+  margin: 0 auto;
+}
+
+h1
+{
+  border-bottom: 1px solid grey;
+
+}
+
+header
+{
+
+  background-color: black;
+  height: 45px;
+}
+
+ul
+{
+  margin-top: 7px;
+}
+nav
+{
```

Comment:
Some of your formatting here is a little hard to follow. Think the most common implementation is

<CSS Selector>{
  <field>: value;
}

etc.